Homework/Marianna/ClassesObjectsInheritance/PersonStudentTeacher/Person.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace PersonStudentTeacher
+{
+    public class Person
+    {
+        public Person(string firstName, string lastName, int age)
+        {
+            FirstName = firstName;
+            LastName = lastName;
+            Age = age;
+        }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public int Age { get; set; }
+        public void Hello()
+        {
+            Console.WriteLine("Hello!");
+        }
+        public virtual void Perform() { }
```

Comment:
Use empty lines between sections! All properties come as the first section, then all come all the methods separated with an empty line!